tests/compile-fail/bit_masks.rs

```diff
@@ -46,8 +46,12 @@ fn main() {
 fn ineffective() {
     let x = 5;
 
-    x | 1 > 2; //~ERROR ineffective bit mask
+    x | 1 > 3; //~ERROR ineffective bit mask
     x | 1 < 3; //~ERROR ineffective bit mask
```

Comment:
`x | 1 > 3` ↔ `x > 3`, also for `x` = `2`. However, line 50 is actually an error. I'll investigate.